packages/gatsby-theme-store/src/sdk/orderForm/controller/orderForm.ts

```diff
@@ -17,6 +17,7 @@ export const fragment = gql`
   fragment OrderFormFragment_orderForm on VTEX_OrderForm {
     id
     items {
+      isGift
```

Comment:
Not related for the contextual pipeline, but necessary so it won't break with @vtex/order-items